api/session_test.go

```diff
@@ -21,6 +21,13 @@ var _ = Describe("Session", func() {
 		session = &Session{bus}
 	})
 
+	Describe("#New", func() {
```

Comment:
We only test constructors by integration, and in this case, testing `New` is forcing you to expose the session URL and extend the `Bus` interface unnecessarily. Instead, add an integration test [here](https://github.com/sclevine/agouti/blob/master/internal/integration/page_test.go#L41) that joins an existing page by joining its WebDriver URL (retrieved using the `WebDriver.URL()` method) and session (retrieved using the `Session.Send()` method with a GET to the `/sessions` endpoint) .